src/qibosoq/qick_programs.py

```diff
@@ -452,7 +452,7 @@ def __init__(
         qpcfg: Config,
         sequence: List[Pulse],
         qubits: List[Qubit],
-        *sweepers: List[Sweeper],
+        sweepers: Tuple[Sweeper],
```

Comment:
```suggestion
        sweepers: Tuple[Sweeper, ...],
```
I guess is a tuple with many sweepers, not a singleton ^^